policyengine_uk/variables/gov/dwp/extended_childcare_entitlement/conditions/extended_childcare_entitlement_income_condition.py

```diff
@@ -0,0 +1,54 @@
+from policyengine_uk.model_api import *
+
+
+class extended_childcare_entitlement_income_requirements(Variable):
```

Comment:
Same with filename, which should always match the variable name
```suggestion
class extended_childcare_entitlement_income_eligible(Variable):
```